### PR TITLE
Chore/update 0323

### PR DESCRIPTION
**Pull request by jrconlin**
## Description

* library changes and updates for 2023 - 03
   * addresses ~ 18 CVE

## Testing

No functional changes.

### CODE REVIEW

**Comment by jbuck** [comment]
I think https://github.com/mozilla-services/channelserver/pull/108/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L19 will need changing to `buster` as well - or maybe it makes more sense to switch to the non-debian-version-specific name?

**Comment by pjenvey** [comment]
I think this branch is out of date with master. Anyway, I had already started the quicker fix to the Dockerfile here: https://github.com/mozilla-services/channelserver/pull/109

File: channelserver/src/server.rs [hunk 1]
```diff
@@ -394,18 +391,8 @@ impl Handler<Connect> for ChannelServer {
...
-        {
-            warn!(
-                self.log.log,
-                "Could not send path to channel";
-                "channel" => chan_id,
-                "remote_ip" => remote
-            )
-        };
+        msg.addr
+            .do_send(TextMessage(MessageType::Text, jpath.to_string()));
```
Comment (pjenvey) [inline channelserver/src/server.rs:395]: Do we want to keep this warn log message as there's now an `Addr::try_send` that returns an Error like the old `do_send` did.

Comment (jrconlin) [reply channelserver/src/server.rs:395]: Ah, sounds good. Forgot that `try_send` was an option

**Review by pjenvey — APPROVED** [review]

**Pull request merged**